src/libraries/System.Security.Cryptography/src/Resources/Strings.resx

```diff
@@ -411,6 +411,9 @@
   <data name="Cryptography_HashLengthMismatch" xml:space="preserve">
     <value>The specified hash is not a valid size for this hash algorithm.</value>
   </data>
+  <data name="Cryptography_HashMLDsaAlgorithmMismatch" xml:space="preserve">
+    <value>The current platform does not support HashML-DSA '{0}' with hash algorithm '{1}'.</value>
```

Comment:
I'm wondering if we should change this to not say "the current platform", but... it's probably fine, even if it really means "all platforms".  Because the current one is one of "all".

Reply:
Eh, yeah, but I still think it should be changed. It... implies... its a platform thing, not an algorithmic requirement.